index.js

```diff
@@ -44,9 +46,19 @@ function requestAsEventEmitter(opts) {
 			res.url = redirectUrl || requestUrl;
 			res.requestUrl = requestUrl;
 
-			if (isRedirect(statusCode) && opts.followRedirect && 'location' in res.headers && (opts.method === 'GET' || opts.method === 'HEAD')) {
+			const followRedirect = opts.followRedirect && 'location' in res.headers;
+			const redirectGet = followRedirect && getMethodRedirectCodes.includes(statusCode);
+			const redirectAll = followRedirect && allMethodRedirectCodes.includes(statusCode);
+
+			if (redirectAll || (redirectGet && ['GET', 'HEAD'].includes(opts.method))) {
```

Comment:
We still support Node.js 4. Just do `.includes(opts.method)` => `.indexOf(opts.method) !== -1`